content/13.conclusion.md

```diff
@@ -1 +1,6 @@
 ## Conclusion {#conclusion}
+
+Deep learning techniques have the potential for wide use in biology, meeting or exceeding the performance of both humans and the current state-of-the art algorithms in an variety of tasks while generating biological insights that may be applied back to fundamental research. 
+To realize this potential, the use of deep learning as a research tool must be approached as any other tool would be: thoughtfully, systematically, and [**TODO**: fill in an adverb here. Ideas: deliberately, ethically, carefully, reproducibly].
```

Comment:
thoughtfully, logically, and systematically?

Reply:
I like the use of "scientifically" somewhere. I like it more than systematically :)

Reply:
Ok what about: "scientifically, thoughtfully, and [adverb]".  "Logically" and "deliberately" seem to be the covered by "thoughtfully" and "reproducibly" is encompassed by "scientifically".

Reply:
What about thoughtfully and scientifically and leave it at that? Good things come in threes, but in this case since we can't figure out the third maybe going with two we're confident in is better 😁

Reply:
Agreed, lets keep it simple. I like using the two.